.github/workflows/build_wamr_vscode_ext.yml

```diff
@@ -32,11 +32,14 @@ jobs:
         working-directory: test-tools/wamr-ide/VSCode-Extension
 
       - name: generate wamr ide vscode extension
+        env: 
+          credentials: ${{ secrets.TOKEN }}
         run: |
           npm install -g vsce
           rm -rf node_modules
           npm install
           vsce package
+          vsce publish -p ${{ secrets.TOKEN }}
```

Comment:
Given we now publish the extension to a market place, I wonder if we'll need some mechanism to check compatibility between VSCode version and WAMR version? It might be a problem already if some users use old version of WAMR (and will be a problem if there will be features added to VSCode extension that make it incompatible with older WAMR versions).

Reply:
Thanks, I've added your email to the email chain, does the link work now ? I made the project private instead of public.

Who would have access to it, unsure of who specifically.

I believe we haven't agreed on the owner of the organization 

I

Reply:
@fadss Many thanks for uploading the PR! Could you help grant the access for `Jun1.Xu@intel.com`, `ning4.wang@intel.com` and `tianlong.liang@intel.com`? They are the main maintainers of WAMR-IDE.

Reply:
Added to the chain, do you have access now ?

Reply:
> Added to the chain, do you have access now ?

Yes, thanks a lot!